examples/ota-provider-app/ota-provider-common/BdxOtaSender.cpp

```diff
@@ -29,6 +30,12 @@ using chip::bdx::StatusCode;
 using chip::bdx::TransferControlFlags;
 using chip::bdx::TransferSession;
 
+bool IsStatusReportMessage(const TransferSession::MessageTypeData & msgTypeData)
```

Comment:
Might be cleaner to add a typed `HasMessageType` accessor on `MessageTypeData`, similar to the way `PayloadHeader` works.  That would automatically handle figuring out the protocol id from the message id.

Reply:
I like this suggestion. Added